### PR TITLE
Add opportunity to filter tests

### PR DESCRIPTION
**Pull request by andreschab90**
With this PR a user can filter tests for his run.

See Docs: http://docs.aws.amazon.com/devicefarm/latest/APIReference/API_ScheduleRunTest.html#devicefarm-Type-ScheduleRunTest-filter

closes https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/issues/39

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -70,6 +70,12 @@ inputs:
       summary: ""
       description: "[See the Test.type documentation](http://docs.aws.amazon.com/devicefarm/latest/APIReference/API_Test.html#devicefarm-Type-Test-type)."
       is_required: true
+  - filter: ""
+    opts:
+      title: "Test Filter"
+      summary: ""
+      description: "[This step adds the opportunity to filter your tests. For example com.android.abc.test1."
```
Comment (fadookie) [inline step.yml]: can you link the relevant docs, scant though they are?
Ex:
```
"This step adds the opportunity to filter your tests. For example com.android.abc.test1. [See the ScheduleRunTest.filter documentation](https://docs.aws.amazon.com/devicefarm/latest/APIReference/API_ScheduleRunTest.html#devicefarm-Type-ScheduleRunTest-filter)."
```

Comment (andreschab90) [inline step.yml]: Done

**Comment by fadookie** [comment]
Thanks, this looks great! I will try to make some time in the next week or two to test this. If it works I will merge & cut a new release.

**Comment by andreschab90** [comment]
thanks ;-)

**Comment by fadookie** [comment]
You had a syntax error in the json that was causing run scheduling to fail. When I saw this I realized there were some serious problems with the error handling in this script as the step was reporting success. I've fixed those and the json syntax error in https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/pull/41

**Pull request merged**